src/transaction/requests.rs

```diff
@@ -547,13 +563,82 @@ pub struct SecondaryLocksStatus {
     pub commit_ts: Option<Timestamp>,
 }
 
-impl HasLocks for kvrpcpb::CommitResponse {}
+impl HasLocks for kvrpcpb::CommitResponse {
```

Comment:
Could you use a macro for these impls, it's a lot of duplicated code.

Reply:
Sure